tests/index.spec.ts

```diff
@@ -18,4 +18,14 @@ describe('jsLogger', function () {
 
     expect(logLine).toHaveProperty('msg', 'avi');
   });
+
+  it('should support base option', function () {
+    const logger = jsLogger({ base: { key: 'value' } }, 'avi-base.log');
```

Comment:
where is the check that the base key | value was written?
i expected a test where the log is parsed and a base log was added

Reply:
done.